This patch removes the clock scaling options from the FPGA configuration dialog.

diff --git a/src/main/java/com/cburch/logisim/fpga/gui/FpgaClockPanel.java b/src/main/java/com/cburch/logisim/fpga/gui/FpgaClockPanel.java
index b8b0bf395..20c80c834 100644
--- a/src/main/java/com/cburch/logisim/fpga/gui/FpgaClockPanel.java
+++ b/src/main/java/com/cburch/logisim/fpga/gui/FpgaClockPanel.java
@@ -66,7 +66,6 @@ public class FpgaClockPanel extends JPanel implements ActionListener, LocaleList
     preDivider.addActionListener(this);
     preDivider.setActionCommand("divider");
     clockScalingPane.add(pan1, BorderLayout.CENTER);
-    add(clockScalingPane, BorderLayout.NORTH);
 
     JPanel pane = new JPanel();
     pane = new JPanel();
